pages/am/3.x/user-guide/user-management/user/user-guide-users.adoc

```diff
@@ -8,11 +8,117 @@
+
+NOTE: Multiple expressions may be combined using the logical operators (`and` / `or`). Operators `[`, `]` and `not` are not supported.
+
+NOTE: In the Portal UI, you don't need to specify the `filter` parameter but only the query value (i.e displayName eq "john doe").
+
+WARNING: By default only the `id`, `externalId`, `username`, `email`, `displayName`, `firstName` and `lastName` fields are indexed in the database.
+This operation supports also to search in the `additionalInformation` attributes of your users, but it's up to you to create the right indexes according to your usages.
+
+The following operators are supported :
+
+.Supported operators
+[cols="1,1,3"]
+|===
+|Operator |Description |Behavior
+
+|eq       |equal        | The attribute and operator values must be identical for a match.
+|ne       |not equal    | The attribute and operator values are not identical.
+|co       |contains     | The entire operator value must be a substring of the attribute value for a match.
+|sw       |starts with  | The entire operator value must be a substring of the attribute value, starting at the beginning of the attribute value. This criterion is satisfied if the two strings are identical.
+|ew       |ends with    | The entire operator value must be a substring of the attribute value, matching at the end of the attribute value.  This criterion is satisfied if the two strings are identical.
+|pr       |present      | If the attribute has a non-empty or non-null value, or if it contains a non-empty node for complex attributes, there is a match.
+|gt       |greater than | If the attribute value is greater than the operator value, there is a match. The actual comparison is dependent on the attribute type. For string attribute types, this is a lexicographical comparison, and for DateTime types, it is a chronological comparison.
+|ge       |greater than or equal to | If the attribute value is greater than or equal to the operator value, there is a match. The actual comparison is dependent on the attribute type. For string attribute types, this is a lexicographical comparison, and for DateTime types, it is a chronological comparison.
+|lt       |less than   | If the attribute value is less than the operator value, there is a match. The actual comparison is dependent on the attribute type. For string attribute types, this is a lexicographical comparison, and for DateTime types, it is a chronological comparison.
+|le       |less than or equal to  | If the attribute value is less than or equal to the operator value, there is a match. The actual comparison is dependent on the attribute type. For string attribute types, this is a lexicographical comparison, and for DateTime types, it is a chronological comparison.
+|===
+
+The following examples will help you to start with this feature. All the attribute names are based on the User model provided by the link:/am/current/management-api/index.html[Rest API].
```

Comment:
The following examples give guidance on how to use this feature. All the attribute names are based on the user model provided by link:/am/current/management-api/index.html[AM API].